perun/cli_groups/import_cli.py

```diff
@@ -31,18 +37,18 @@
     help="Specifies the head minor version, for which the profiles will be imported.",
 )
 @click.option(
-    "--exitcode",
-    "-e",
+    "--stats-info",
+    "-t",
     nargs=1,
-    required=False,
-    default="?",
-    help=("Exit code of the command."),
+    default=None,
+    metavar="<stat1-description,...>",
```

Comment:
```suggestion
    metavar="[STAT]",
```
In my WIP I'm rewritting the metavars to conform to `[...]` notation. Please, use this.